### DISCUSSION TITLE
I opened a pr 7 days ago, there is still no feedback or suggest

### DISCUSSION THREAD

**Discussion by wqs576222103** [type-challenges/type-challenges#12988] [Q&A] [answered]
the github-actions did not handle it. hope someone to review it [#12776](https://github.com/type-challenges/type-challenges/pull/12776)

**Comment by JonghwanWon** [comment, marked as answer]
Hi, @wqs576222103 

It seems that you need to use a [new-challenges](https://github.com/type-challenges/type-challenges/issues/new?assignees=&labels=new-challenge&template=0-new.md&title=) in the issue template to create new challenge.
